src/Wig.Zstd/Program.cs

```diff
@@ -0,0 +1,190 @@
+﻿namespace Wig.Zstd
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection.Emit;
+    using System.Reflection.Metadata;
+    using System.Reflection.Metadata.Ecma335;
+    using System.Runtime.Serialization;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Spectre.Console;
+    using Spectre.Console.Cli;
+    using ZstdNet;
+
+    public static class Program
+    {
+        public static async Task<int> Main(string[] args)
+        {
+            var app = new CommandApp<DefaultCommand>();
+            app.Configure(config =>
+            {
+                config.SetApplicationName("dotnet example");
+            });
+
+            return await app.RunAsync(args);
+        }
+    }
+
+    public sealed class DefaultCommand : AsyncCommand<DefaultCommand.Settings>
+    {
+        private readonly IAnsiConsole _console;
+
+        public sealed class Settings : CommandSettings
+        {
+            [CommandArgument(0, "path")]
+            [Description("Path to the files")]
+            public string Path { get; set; }
+
+            [CommandOption("-c|--compress")]
+            [Description("Compress files")]
+            public bool IsCompressionMode { get; set; }
+
+            [CommandOption("-d|--decompress")]
+            [Description("Decompress files")]
+            public bool IsDecompressionMode { get; set; }
+
+            [CommandOption("-o|--overwrite")]
+            [Description("Overwrite file / directory")]
+            public bool Overwrite { get; set; }
+
+            [CommandOption("-l|--level")]
+            [Description("Compression level (1-22)")]
+            public int CompressionLevel { get; set; }
+
+            [CommandOption("-s|--subfolder")]
```

Comment:
I think this should be a boolean flag. If the user, uses this option then the command will look into subfolders. Otherwise if the command is not used it will not look into subfolders